src/swarm/peers.js

```diff
@@ -25,63 +29,37 @@ module.exports = (createCommon, options) => {
 
     after(() => common.teardown())
 
-    it('should list peers this node is connected to', (done) => {
-      ipfsA.swarm.peers((err, peers) => {
-        expect(err).to.not.exist()
-        expect(peers).to.have.length.above(0)
+    it('should list peers this node is connected to', async () => {
+      const peers = await ipfsA.swarm.peers()
+      expect(peers).to.have.length.above(0)
 
-        const peer = peers[0]
+      const peer = peers[0]
 
-        expect(peer).to.have.a.property('addr')
-        expect(multiaddr.isMultiaddr(peer.addr)).to.equal(true)
-        expect(peer).to.have.a.property('peer')
-        expect(PeerId.isPeerId(peer.peer)).to.equal(true)
-        expect(peer).to.not.have.a.property('latency')
+      expect(peer).to.have.a.property('addr')
+      expect(multiaddr.isMultiaddr(peer.addr)).to.equal(true)
+      expect(peer).to.have.a.property('peer')
+      expect(PeerId.isPeerId(peer.peer)).to.equal(true)
+      expect(peer).to.not.have.a.property('latency')
 
-        // only available in 0.4.5
-        // expect(peer).to.have.a.property('muxer')
-        // expect(peer).to.not.have.a.property('streams')
-
-        done()
-      })
+      // only available in 0.4.5
```

Comment:
I'm guessing this refers to `go-ipfs@0.4.5`?  It's on `0.4.22` at the moment so these assertions can probably be uncommented?